htsohm/htsohm.py

```diff
@@ -396,6 +396,9 @@ def worker_run_loop(run_id):
             material.generation_index = material.calculate_generation_index()
             if material.generation_index < config['children_per_generation']:
                 session.add(material)
+            elif material.generation_index == config['children_per_generation'] - 1:
+                # DO STUFF
+
```

Comment:
If you were going to placeholder this, you have to use pass; but this doesn't need to be placeholdered. These last three lines should be squashed with next commit.